utils-intervalrdd/src/main/scala/org/bdgenomics/utils/interval/array/IntervalArray.scala

```diff
@@ -193,10 +265,10 @@ trait IntervalArray[K <: Interval[K], T] extends Serializable {
   }
 
   /**
-   * Insert an Iterator of (K,V) items into existing IntervalArray.
+   * Insert an Iterator of (K,T) items into existing IntervalArray.
```

Comment:
Since we doc the `K` and `T` types in the trait scaladoc, I would drop the `(K,T)` here.